docs/services/topology.md

```diff
@@ -83,20 +97,44 @@ All changes should be tested on the ITB instance before deploying to production.
 If you can, test them on your local machine first.
 These instructions assume that the code has not been merged to master.
 
-1.  Update the Git clone at `/opt/topology-itb`:
+1.  Update the ITB software installation at `/opt/topology-itb`:
 
         :::console
         # cd /opt/topology-itb
         # git fetch --all
```

Comment:
Say something about adding your repo as a remote if necessary. Just use the instructions from 125--137. The `git checkout` command should also be changed to match 132.